src/stylesheets/elements/form-controls/_text-input.scss

```diff
@@ -10,6 +10,12 @@
 
 .usa-input--error {
   @include u-border($theme-input-state-border-width, "error-dark");
+  padding: calc(#{units(1)} - #{units($theme-input-state-border-width)});
+}
+
+.usa-input--success {
+  @include u-border($theme-input-state-border-width, "success-dark");
+  padding: calc(#{units(1)} - #{units($theme-input-state-border-width)});
 }
```

Comment:
Not sure why it's in a different place, but `.usa-input--success` is in `_global.scss` too: https://github.com/uswds/uswds/blob/develop/src/stylesheets/elements/form-controls/_global.scss#L26-L28

Reply:
Good point. We'll have to move them to where they make most sense, thank you!

Reply:
@mahoneycm let's move these error/success styles to `form-controls/_global.scss`